languagetool-language-modules/pt/src/main/resources/org/languagetool/rules/pt/style.xml

```diff
@@ -9914,6 +9914,49 @@ USA
         </rulegroup>
 
 
+        <rule id='FAZER_EXERCER' name="[Universitário][Científico] Verbo Fazer → Verbo Exercer" type="style" tone_tags="academic" is_goal_specific="true" tags="picky" default="temp_off">
+            <antipattern>
+                <token postag='V.+' postag_regexp='yes'/>
+                <token min='0' max='1' postag='D[AI].+|NC.+|AQ.+' postag_regexp='yes'/>
+                <token inflected='yes'>atividade</token>
+                <token inflected='yes'>físico</token>
+                <example>Conclusão, estava em jejum e fazendo atividade física pesada.</example>
+                <example>Faça atividades físicas todos os dias.</example>
+                <example>Tom nunca faz atividades físicas.</example>
+            </antipattern>
+            <antipattern>
+                <token postag='V.+' postag_regexp='yes'/>
+                <token min='0' max='1' postag='D[AI].+|NC.+|AQ.+' postag_regexp='yes'/>
+                <token inflected='yes'>força</token>
+                <token regexp='yes' inflected='yes'>&languages;|aéreo|armada|espacial|marítimo|militar|naval|terrestre</token>
+                <example>Com o objetivo de fazer as forças alemãs acreditarem que a invasão aliada viria de Pas-de-Calais, a força fantasma foi esta...</example>
+                <example>O país vai fazer as forças terrestres deslocarem-se.</example>
+                <example>É isso que fazem as forças espaciais.</example>
+            </antipattern>
+            <antipattern>
+                <token postag='V.+' postag_regexp='yes'/>
+                <token postag='CC' skip='2'/>
+                <token inflected='yes'>fazer</token>
+                <token min='0' max='1' postag='D[AI].+|NC.+|AQ.+' postag_regexp='yes'/>
+                <token inflected='yes'>força</token>
+                <example>...ilhas ou facetas de desgaste nos dentes e tem uma alavancagem mandíbula que não é projetado para as fazer muita forças para cortar e triturar.</example>
+                <example>O médico disse para não fazer força.</example>
+                <example>O médico disse para eu não fazer força.</example>
+            </antipattern>
+            <pattern>
+                <marker>
+                    <token inflected='yes'>fazer</token>
+                </marker>
+                <token min='0' max='1' postag='D[AI].+|NC.+|AQ.+' postag_regexp='yes'/>
+                <token regexp='yes' inflected='yes'>atividade|cargo|força|influência|pressão|profissão|tensão|terror|violência</token> <!-- Add words here when they are found -->
```

Comment:
Got this from a corpus search for collocations with `exercer`, could server as inspiration.

![Screenshot 2023-10-09 at 9 48 56 AM](https://github.com/languagetool-org/languagetool/assets/50704700/de065732-5bef-44b1-ac0f-1a7f8673f023)